src/main/java/com/ericsson/ei/subscription/RunSubscription.java

```diff
@@ -101,10 +100,11 @@ public boolean runSubscriptionOnObject(String aggregatedObject, Iterator<JsonNod
                 boolean resultNotEqualsToNull = !resultString.equals("null");
                 boolean resultNotEqualsToFalse = !resultString.equals("false");
                 boolean resultNotEmpty = !resultString.equals("");
-                LOGGER.debug("Jmespath rule result: '" + result.toString() + "'\nConditions fulfillment:"
-                        + "'\nResult not equals to null' is '" + resultNotEqualsToNull
-                        + " '\nResult not equals to false' is '" + resultNotEqualsToFalse
-                        + "' '\nResult not empty' is '" + resultNotEmpty + "'");
+                LOGGER.debug("Jmespath rule result: '{}'\nConditions fulfillment:"
+                        + "'\nResult not equals to null' is '{}"
```

Comment:
I really don't like these messages,,, when looking at these messages in the logs, user don't understand really what each "result" messages says. Should be improved with more info about each result line and what has not matched/failed.

Reply:
I changed the logs to display something like this:
Condition: { id = "Mighty Mouse" }
JMESPath evaluation result: true
Condition was fulfilled.
It shows which condition it tried to evaluate, shows the result of the evaluation and instead of printing the 3 boolean checks it prints a string in clear text stating if it was fulfilled or not.